src/event.rs

```diff
@@ -1071,19 +1167,41 @@ mod tests {
                 with_window_event(HoveredFile("x.txt".into()));
                 with_window_event(HoveredFileCancelled);
                 with_window_event(Ime(Enabled));
-                with_window_event(CursorMoved { device_id: did, position: (0, 0).into() });
+                with_window_event(PointerMoved {
+                    device_id: did,
```

Comment:
I start to wonder whether device ID should belong here in general, since what kind of device id Touch input has? Maybe it's a property of a specific input device, like `Mouse(DeviceId)`?

Reply:
I know at least one use-case: if your drawing tablet has touch support and your screen as well, these would be two different devices.